src/vendedores.test.ts

```diff
@@ -30,5 +76,95 @@ describe("Vendedores", () => {
       });
     });
+    describe("Comercio Corresponsal", () => {
+      it("en una ciudad donde tenga sucursal", () => {
+        expect(comercioCorresponsal.puedeTrabajarEn(sierra)).toBeTruthy();
+      });
+
+      it("en una ciudad donde NO tenga sucursal", () => {
+        expect(comercioCorresponsal.puedeTrabajarEn(rosario)).toBeFalsy();
+      });
+    });
+  });
+
+  describe("Versatilidad", () => {
+    it("menos de 3 certificaciones no es versátil", () => {
+      expect(vendedorFijo.esVersatil()).toBeFalsy();
+    });
+    it("3 o más certificaciones pero todas del mismo tipo", () => {
+      expect(viajante.esVersatil()).toBeFalsy();
+    });
+    it("3 o más certificaciones, al menos 1 de producto, al menos 1 de NO producto", () => {
+      expect(comercioCorresponsal.esVersatil()).toBeTruthy();
+    });
+  });
+
+  describe("Firmeza", () => {
+    it("menos de 30 puntos no es firme", () => {
+      expect(vendedorFijo.esFirme()).toBeFalsy();
+    });
+    it("30 puntos es firme", () => {
+      expect(viajante.esFirme()).toBeTruthy();
+    });
+    it("más de 30 puntos es firme", () => {
+      expect(comercioCorresponsal.esFirme()).toBeTruthy();
+    });
+  });
+  describe("Influencia", () => {
+    describe("Vendedor Fijo", () => {
+      it("si es vendedor fijo no es influyente", () => {
+        expect(vendedorFijo.esInfluyente()).toBeFalsy();
+      });
+    });
+    describe("Viajante", () => {
+      it("la población de las provincias es superior a 10.000.000", () => {
+        expect(viajante2.esInfluyente()).toBeTruthy;
+      });
+      it("la población de las provincias es inferior a 10.000.000", () => {
+        expect(viajante.esInfluyente()).toBeFalsy;
+      });
+    });
+    describe("Comercio Corresponsal", () => {
+      it("está en 5 o más ciudades", () => {
+        expect(comercioCorresponsal2.esInfluyente()).toBeTruthy();
+      });
+      it("está en menos de 5 ciudades pero en 3 o más provincias", () => {
+        expect(comercioCorresponsal3.esInfluyente()).toBeTruthy();
+      });
+      it("está en menos de 5 ciudades y/o 3 provincias", () => {
+        expect(comercioCorresponsal.esInfluyente()).toBeFalsy();
+      });
+    });
+  });
+});
+
+describe("Centros de distribución", () => {
+  it("su vendedor estrella es quién tenga mayor puntaje", () => {
+    console.log(
+      "Vendedor 0 puntaje: " + centroDeDistribucion.vendedores[0].puntaje()
+    );
+    console.log(
+      "Vendedor 1 puntaje: " + centroDeDistribucion.vendedores[1].puntaje()
+    );
+    console.log(
+      "Vendedor 2 puntaje: " + centroDeDistribucion.vendedores[2].puntaje()
+    );
+    expect(centroDeDistribucion.vendedorEstrella()).toEqual(
+      centroDeDistribucion.vendedores[1]
+    );
+  });
+  it("puede cubrir ciudad si por lo menos un vendedor puede trabajar allí", () => {
+    expect(centroDeDistribucion.puedeCubrir(marDelPlata)).toBeTruthy();
+    expect(centroDeDistribucion.puedeCubrir(rosario)).toBeFalsy();
+  });
```

Comment:
Sería mejor abrir en dos `it`, para que si falla se entienda qué caso no funciona.